### PR TITLE
Add Unit testing translation for PT-BR and ES

### PR DESCRIPTION
**Pull request by vitormalencar**
Add Unit testing translation for PT-BR and ES

### CODE REVIEW

**Comment by vitormalencar** [comment]
hey @joaolucasl can you review for me =) ?

File: content/lang/pt-br/guide/unit-testing-with-enzyme.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        "react": "preact-compat-enzyme",
+        "react-dom": "preact-compat-enzyme"
+    }
+  }
+}
+```
+
+## Limitações atuais
+
+1. No momento, apenas módulo [`mount`](http://airbnb.io/enzyme/docs/api/mount.html)  é suportado.
```
Comment (joaolucasl) [inline content/lang/pt-br/guide/unit-testing-with-enzyme.md]: 'apenas _o_ módulo'

Comment (vitormalencar) [inline content/lang/pt-br/guide/unit-testing-with-enzyme.md]: you are right ! hahahaahh
I gonna fix this =D

**Comment by joaolucasl** [comment]
Cool! But it seems like several of your commits do not appear as changes - were they merged already or? (Perhaps a simple squash will do, but don't quote me on that)

**Comment by developit** [comment]
I copied the translated titles to the frontmatter `name` field.  I'm not sure if it would do anything in the current site, but I'm going to be auto-generating the menu soon so that it takes these into account and is nicely localized :)

Everything here looks good to me BTW.

**Comment by vitormalencar** [comment]
Awesome 🙌 
once again thank you guys  🙇

**Comment by developit** [comment]
nono thank *you*, it's fantastic to have these translations!!

**Pull request merged**